.github/workflows/countline.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+import os
+import glob
+
+#get current working dir, set count, and select file delimiter
+path = os.getcwd()
+#os.chdir(path)
+
+
+#parses through files and saves to a dict
+names={}
+pathName = path + '/**/*.dart'
```

Comment:
Use `os.path.split` and `os.path.join` to create the path. If this is run on a windows system it will fail. Also use `os.path.expanduser` to limit the unlikely case where someone uses a ~ in the path name.

Reply:
Done